examples/PerfectPowers.jl

```diff
@@ -229,13 +229,27 @@ function _root_exact(a::fmpz, p::Int, extra_s::Int = 5, extra_w::Int = 1)
   return D
 end
 
-function ispower_bernstein(a::fmpz)
+function is_power_bernstein(a::fmpz)
```

Comment:
Perhaps also add at least a trivial comment saying what this function does / what its return type is? Bonus points if it is a doc string with a jldoctest ;-)